packages/sn-filter-pane/src/ext/property-panel/data/data-panel/presentation/index.ts

```diff
@@ -106,6 +106,36 @@ export default function getPresentation(env: IEnv) {
           },
         },
       },
+      collapseModeGroup: {
+        type: 'items',
+        grouped: true,
+        show() {
+          return env?.flags?.isEnabled('IM_4072_FILTERPANE_SETTINGS');
+        },
+        items: {
+          collapseMode: {
+            ref: 'collapseMode',
```

Comment:
@veinfors Should we move this to `layoutOptions`, i.e. `layout.layoutOptions.collapseMode`?